packages/mdl-snackbar/README.md

```diff
@@ -0,0 +1,179 @@
+# MDL Checkbox
```

Comment:
Looks like this still contains the copy for `mdl-checkbox`. However, feel free to hold off on changing it until after the code in the PR is reviewed.